doc/news/DM-48022.feature.rst

```diff
@@ -0,0 +1,6 @@
+Deprecate `ignore_m1m3` property.
+Added new property `disable_m1m3_force_balance` with default `false`.
+    Maintains the ability to disable the M1M3 balance system, in case
```

Comment:
I think you can remove this space here before each line. Also, we use "semantic line breaking" so please, make sure each phrase is in one line.